content/plugins/descriptions/api-docs.yaml

```diff
@@ -0,0 +1,65 @@
+# The server must be restarted to pick up changes in ths file.
+
+# Required. Must be unique across all files in the directory. No whitespace. Use hyphens as
+# separators.
+name: api-docs
+humanName: API Docs
+heading: 'Backstage API Docs Plugin'
+# Keep it short
+lead: 'Discover and display API entities for your components in Backstage'
+
+seo:
+  # Don't forget to end with "| Roadie"
+  title: 'Backstage AWS Lambda Plugin | Roadie'
+  description: |
+    Components to discover and display API entities as an extension to the catalog plugin.
+
+logo:
+  # This must be a relative path. It should start without a slash or with ./
+  # The overall size of the file should be approximately 200 by 200.
+  # The image in the file should be approximately 100 by 100 pixels. It needs space around it.
+  # The image will be made greyscale by Gatsby Image Sharp.
+  fileSystemPath: './content/assets/logos/api-docs/logo-docs.png'
```

Comment:
The grey color in this logo is clashing with the green background.

![Screenshot 2020-11-04 at 12 28 19](https://user-images.githubusercontent.com/562403/98111942-44eacc00-1e99-11eb-8103-f95f3bbc479a.png)

Is it possible to edit the logo image to make the grey section transparent?